pkg/cli/doctor.go

```diff
@@ -365,23 +366,50 @@ func fromZipDir(
 	_ = builtins.AllBuiltinNames()
 
 	descTable = make(doctor.DescriptorTable, 0)
-	if err := slurp(zipDirPath, "system.descriptor.txt", func(row string) error {
-		fields := strings.Fields(row)
-		last := len(fields) - 1
-		i, err := strconv.Atoi(fields[0])
-		if err != nil {
-			return errors.Wrapf(err, "failed to parse descriptor id %s", fields[0])
-		}
+	if checkIfFileExists(zipDirPath, "system.descriptor.txt") {
```

Comment:
hmm...this is sad. I wonder what made us missing the change of this part at the first place. Maybe the test should have been more end-to-end. It should have generated the debug zip from a test db instead of having the fixtures in the format it was expecting. But this is low priority and can be future refactoring.

Reply:
Yes, let good point let me add this in a future PR using roachtest